tests/by-util/test_wc.rs

```diff
@@ -42,7 +42,7 @@ fn test_utf8() {
         .args(&["-lwmcL"])
         .pipe_in_fixture("UTF_8_test.txt")
         .run()
-        .stdout_is("   300  4969 22781 22213    79\n");
+        .stdout_is("  300  4969 22781 22213    79\n");
     // GNU returns "  300  2086 22219 22781    79"
```

Comment:
Is this comment/todo still relevant?

Reply:
Yes, the TODO is still relevant because the counts for words, characters, and bytes differ. This pull request eliminates the leading whitespace, but it does not fix any of the incorrect counts.

Reply:
I created issue #2185 to represent that issue.